sfdx/README.md

```diff
@@ -16,19 +16,19 @@ A plugin for the Salesforce CLI to create mobile applications to interface with
 
 ### Install as plugin
 
-1. Install plugin: `sfdx plugins:install sfdx-mobilesdk-plugin`
+1. Install the plugin: `sfdx plugins:install sfdx-mobilesdk-plugin`
 
 ## Help
 ```
 -> sfdx mobilesdk --help
 Usage: sfdx COMMAND [command-specific-options]
 
-Help topics, type "sfdx help TOPIC" for more details:
+Help topics. Type "sfdx help TOPIC" for more details.
```

Comment:
That line is coming from sfdx, we can't change it.

Reply:
Can't change it because you will regenerate this content by running sfdx commands? If not, I have no qualms about fixing their grammatical and punctuation errors in the readme. Maybe it will prompt them to fix the help strings in the script itself.